nemo/collections/common/prompts/canary.py

```diff
@@ -18,7 +18,7 @@ class CanaryPromptFormatter(PromptFormatter):
                 "source_lang": Modality.Text,
                 "task": Modality.Text,
```

Comment:
```suggestion
                "task": Modality.TextLiteral("asr", "s2t_translation", "<|transcribe|>", "<|translate|>"),
```

Reply:
I think it was either "asr" or "s2t_translation" for some reason.

Reply:
Ill add all three just to be safe